src/Symfony/Component/Serializer/NameConverter/MetadataAwareNameConverter.php

```diff
@@ -47,7 +47,7 @@ public function normalize($propertyName, string $class = null, string $format =
             return $this->normalizeFallback($propertyName, $class, $format, $context);
         }
 
-        if (!isset(self::$normalizeCache[$class][$propertyName])) {
+        if (!\array_key_exists($class, self::$normalizeCache) && !\array_key_exists($propertyName, self::$normalizeCache[$class])) {
```

Comment:
|| instead of &&? the first could be an isset for readability?

Reply:
correct.